content/en/opentelemetry/agent/install_agent_with_collector.md

```diff
@@ -82,9 +68,11 @@ Use a YAML file to specify the Helm chart parameters for the [Datadog Agent char
 1. Configure the Datadog API and application key secrets:
    {{< code-block lang="yaml" filename="datadog-values.yaml" collapsible="true" >}}
 datadog:
+  site: {{< region-param key="dd_site" >}}
   apiKeyExistingSecret: datadog-secret
   appKeyExistingSecret: datadog-secret
    {{< /code-block >}}
+   Set `datadog.site` to your [Datadog site][52]. Otherwise, it defaults to US1.
```

Comment:
```suggestion
   Set `datadog.site` to your [Datadog site][52]. Otherwise, it defaults to `datadoghq.com`, the US1 site.
```
I think the previous sentence could potentially be misinterpreted as the default value being `US1`. What do you think of this adjustment?